lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_consumer.erl

```diff
@@ -145,6 +145,16 @@ on_start(InstanceId, Config) ->
                 instance_id => InstanceId,
                 kafka_hosts => BootstrapHosts
             });
+        {error, already_present = Reason} ->
+            ?SLOG(error, #{
+                msg => "failed_to_start_kafka_consumer_client",
+                instance_id => InstanceId,
+                kafka_hosts => BootstrapHosts,
+                reason => emqx_misc:redact(Reason)
```

Comment:
nit: no need to redact ?

Reply:
:joy_cat: 

https://github.com/emqx/emqx/pull/9564#discussion_r1128570431

Reply:
the `Reason` here in this clause is obviously an atom.
The next clause however might contain sensitive info.